### PR TITLE
Add SSD1322_rs crate to the list

### PR DESCRIPTION
**Pull request by birdistheword96**

### CODE REVIEW

**Review by BartMassey — CHANGES_REQUESTED** [review]
The spelling still isn't right :grinning:

**Review by BartMassey — COMMENTED** [review]
I'm not sure how we're doing stuff, but a little tl;dr about the differences between `ssd1322_rs` and `ssd1322` (immediately above) and which one might choose to use in what situations would be super-helpful to put somewhere here. Maybe just a few words in parentheses would be sufficient?

**Comment by birdistheword96** [comment]
Thanks for the feedback, I wasn't sure what else to put on how to distinguish which ssd1322 crate to use other than it's async (the other one is blocking) which I have included in the description.  Didn't want to make it too wordy! 

Hopefully that's enough but happy to make any other changes.

**Review by BartMassey — APPROVED** [review]
Yeah, I was being dumb about the sync vs async thing — it's obvious. Apologies.

This looks great! Thanks much.
